utils/jwtUtil.js

```diff
@@ -19,10 +19,52 @@ const User = require('../db/index').User;
 const Token = require('../db/index').Token;
 const Collector = require('../db/index').Collector;
 const Bot = require('../db/index').Bot;
+const Profile = require('../db/index').Profile;
 const Promise = require('bluebird');
 const jwtVerifyAsync = Promise.promisify(jwt.verify);
 
 /**
+ * Adds a key and its value to the passed in object.
+ * @param  {Object} object - The object to which the key value pair needs to be
+ * assgined
+ * @param  {String} key - The key that will be added to the object
+ * @param  {String} value  - The value that will be assigned to the key
+ */
+function assignKeyValue(object, key, value) {
+  object[key] = value;
+} // assignKeyValue
+
+/**
+ * Assigns user related information to the request header.
+ * @param  {Object}  req  - The request object
+ * @param  {String}  tokenName - Token name parsed from the token
+ * @param  {Boolean} isAdmin  - Flag to indicate if the request was made by an
+ * Admin user
+ */
+function assignUserHeaders(req, tokenName, isAdmin) {
+  assignKeyValue(req.headers, 'userName', req.user.name);
```

Comment:
I had intentionally written these header names up in the story as starting with upper-case letters so that they would *not* look like variable names, since they're actually internal constants... what was your rationale for making them start with lower-case? Should we prefix then with "Refocus" like RefocusTokenName, RefocusIsBot, etc.? Or change them to TOKEN_NAME, PROFILE_NAME, USER_NAME, etc. so they look more like constant names? Thoughts?

Reply:
Missed it!! I can change it to start with an upper case. I like the way it is defined in the story.